XenModel/Messages.resx

```diff
@@ -14377,6 +14395,9 @@ Are you sure you want to enable automated power management for this Host?</value
   <data name="XENSERVER_8_1" xml:space="preserve">
     <value>[XenServer] {0}</value>
   </data>
+  <data name="XENSERVER_8_2" xml:space="preserve">

```

Comment:
Can we use a generic message here, e.g. XENSERVER_VERSION = [XenServer] {0},
to replace XENSERVER_8_1 and XENSERVER_8_2?

Reply:
Actually, since the product name-version is the same in non-English locales, I was thinking of removing them completely from the Messages.resx and moving them as XENSERVER_8_1 ="Citrix Hypervisor 8.1", XENSERVER_8_2="Citrix Hypervisor 8.2" to Branding.resx. They would replace the PRODUCT_VERSION_xxx constants because I don't think we use the numbers alone anywhere and we can avoid nested string.Format() calls.

Reply:
In any case, I'll leave this change for a different PR on master (as the last installment of CP-31105).